math/maxima/Portfile

```diff
@@ -88,4 +115,10 @@ variant printable_doc description {Build printable documentation} {
     }
 }
 
-livecheck.regex  {<title>.*/Maxima-source/(.*)-source/maxima.*</title>}
+notes   "
+    Maxima includes the following commands: \n\
```

Comment:
Only one space after `notes` is our convention. And I personally prefer not indenting the note.